### PR TITLE
Fixed duplicate key error handling for multivalue insert.

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java [hunk 1]
```diff
@@ -317,6 +318,26 @@ public void testInsertWithPrimaryKey() throws Exception {
...
+        refresh();
+
+        expectedException.expect(SQLActionException.class);
+        expectedException.expectMessage("DuplicateKeyException: A document with the same primary key exists already");
+        args = new Object[]{"1",
+            "I always thought something was fundamentally wrong with the universe."};
+        execute("insert into test (pk_col, message) values (?, ?)", args);
+        refresh();
+
+        GetResponse response = client().prepareGet("test", "default", "1").execute().actionGet();
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java]: I think this won't get executed if the execute before throws the exception

File: sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java [hunk 2]
```diff
@@ -729,6 +760,30 @@ public void testBulkInsertWithMultiValue() throws Exception {
     }
 
     @Test
+    public void testBulkInsertWithMultiValueFailing() throws Exception {
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java]: isn't there already a test for exactly this case?

File: sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java (hunk 1 above)
Comment (seut) [inline sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java]: pls don't use ES api here, no need to do so. also afaik its not needed to validate that the inserto did not ocurre

File: sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java (hunk 2 above)
Comment (matriv) [inline sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java]: This is a bulk insert with multivalues. We have one with multivalues but not bulk and one with bulk but not multivalues. Do I miss something?

**Comment by mfussenegger** [comment]
lgtm

**Pull request merged**